converter/lookml/lookml.py

```diff
@@ -0,0 +1,87 @@
+from itertools import takewhile
+import os
+import lkml
+from typing import Iterator, List, Optional
+
+from dbt_semantic_interfaces.parsing.dir_to_model import SemanticManifestBuildResult
+from dbt_semantic_interfaces.implementations.semantic_model import PydanticSemanticModel
+from dbt_semantic_interfaces.type_enums.time_granularity import TimeGranularity
+from dbt_semantic_interfaces.implementations.semantic_manifest import (
+    PydanticSemanticManifest,
+)
+from dbt_semantic_interfaces.implementations.semantic_model import NodeRelation
+
+from tqdm import tqdm
+
+from converter.lookml.dimension import to_sl_dimensions
+from converter.lookml.measure import to_sl_measures
+from converter.lookml.model import LkmlModel
+
+DEFAULT_TIME_GRANULARITY = TimeGranularity.DAY
```

Comment:
You should be able to import this from MetricFlow instead of redefining it here, right?